module-4/backend/week_one.md

```diff
@@ -1,21 +1,49 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's the most useful thing you learned from completing the intermission week work?
+basic JS syntax/vocabulary - it helps me talk through problems, and set up simple code (that can be then be addressed/lends to my google-fu)
+
 2. What are some tools to help debug JavaScript code?
+dev tools: source code/debugger/console/element/network
+
 3. What are some tools you need in order to unit test your JavaScript?
+mocha/chai
+
 4. What is the syntax for invoking a function?
+functionName() - the parentheses at the end
+
 5. What is CORS?
+Cross Origin Resource Sharing
+it's a mechanism that allows you to HTTP resources from a separate domain than the domain from which your app was loaded
+
 6. How do we enable CORS?
+add an HTTP headers tag? I still need to enable - haven't pushed to production
+
 7. What's `this` in JavaScript?
+the entirety of whatever context you're working in - and all of the prototypes accessible to you for that context
+
 8. What is Webpack and why is it useful?
+a module bundler - it bundles all of the 'make it pretty' files in order to export them to your browser (it's a wannabe asset pipeline)
+
 9. When/why do you want to use event delegation?
+when we set up an event listener to a listen/react to a particular event
```

Comment:
This describes more of what an event listener is than event delegation. Event delegation allows us to take advantage of event bubbling to handle events that may happen on elements that currently exist in the dom or may be added in the future. We can add an event to a parent element and event bubbling will always cause our event to be triggered whether the child element existed on page load or not.